app/views/certificates/_certificates_table.html.erb

```diff
@@ -16,9 +16,10 @@
       <td class="govuk-table__cell"><%= certificate.usage %></td>
       <td class="govuk-table__cell"><%= certificate.enabled %></td>
       <td class="govuk-table__cell">
-        <%= form_for [@component, certificate], method: :patch do |f| %>
-          <%= f.hidden_field :enabled, value: !certificate.enabled %>
-          <%= f.submit certificate.enabled ? 'Disable' : 'Enable', class: 'govuk-button' %>
+        <%= form_for [@component, certificate],
+          method: :patch,
+          url: "/components/#{@component.id}/certificates/#{certificate.id}/#{status}"  do |f| %>
```

Comment:
you should be able to use a url_helper for this

Reply:
It can but not sure if this is really worth it. Had to do the following
```
  def toggle_enable_on_button(status, component, certificate)
    helper = Rails.application.routes.url_helpers
    name = "#{status}_component_certificate_path"
    helper.public_send(name, component, certificate) if helper.respond_to?(name)
  end
```
to get this to work